templates/template/packages/README.md

```diff
@@ -82,38 +80,35 @@ Developers will use the following commands to work with packages:
+  - Note: On a prepare, the charts-build-scripts will automatically replace your charts existing dependencies with dependencies that will show up in `generated-changes/dependencies/<dependency>/dependency.yaml`. Since the spec of this file follows the `UpstreamConfiguration` described above, you can modify this to point your dependencies to a local chart (rooted at `generated-changes/dependencies/<dependency>/`), another package, a Chart archive (e.g. point it to a newer version of the dependency), or a Github Repository (at a subdirectory / commit).
   - Note: As part of replacing your dependencies, the Chart.yaml and requirements.yaml are considered "Managed Files". These files are prone to having conflicts and may need to be deleted / recreated on a prepare if there are conflicts. Please open up an issue if you encounter frequent bugs with these files so that we can take a look.
 - Make changes to the relevant working directories
 - Run `PACKAGE=<package> make patch` to save your changes
-- Run `make charts` to test your changes by committing the generated directories and pushing it to a branch. Once it is available at such a branch, it can be picked up by any tools that can point to Github based Helm Repositories (e.g. Rancher Cluster Explorer Apps & Marketplace).
-- Repeat above steps if you encounter bugs, otherwise continue.
-- Remove the commit with your newly generated resources (`assets/`, `charts/` and `index.yaml`) and remove the resources themselves manually
 - Run `make clean`
-- Commit your changes and push them to another branch to get ready for a PR
-- Run `make validate` to ensure that your current repository wouldn't introduce any conflicts on a sync. This step can be skipped if a Github Workflow already checks for this on a PR push
-- Open up a PR with your changes
+- Run `make charts` to test your changes by committing the generated directories and pushing it to a branch. Once it is available at such a branch, it can be picked up by any tools that can point to Github based Helm Repositories (e.g. Rancher Cluster Explorer Apps & Marketplace).
+- If you encounter bugs, undo the `make charts` commit (e.g. `git reset HEAD~1 --hard` or `git reset HEAD~1 --soft; git checkout -- .; git clean -df` if you want to do it in multiple steps) and repeat the above steps.
```

Comment:
Or ammend and push force right, nothing special here about undoing commit.

Reply:
yeah nothing special. I just like to delete the commit entirely though and recreate 😄 .

Should this be changed?

Reply:
Its fine, but I would just say in general be conscious of how much the person has to read. If you said "force push or undo the last commit" you'd have the same affect. Right now they have to read the commands and figure out if something special is going on. For git status thats not a big deal, for longer commands and sequences its quite taxing.

Reply:
Updated the working for this, made it slightly more verbose but easier to read with no commands.